ui/src/apps/logSearching/components/SearchProgress.tsx

```diff
@@ -207,14 +200,14 @@ export default function SearchProgress({
         name === key
       )
     )
-    
+
     const res = await client.dashboard.logsDownloadAcquireTokenGet(keys)
     const token = res.data
     if (!token) {
       return
     }
     const url = `${DASHBOARD_API_URL}/logs/download?token=${token}`
-    downloadFile(url)
+    window.open(url)
```

Comment:
Open a new window may be intercepted by some browser, I suggest download file by creating a hidden `<a>` element and click it.

Reply:
If opening a new window is intercepted, how will this method work?

Reply:
How about this solution? https://stackoverflow.com/questions/19026162/javascript-window-open-from-callback Looks like this should be the correct answer.

Reply:
OK for me.